arduino/smartcar/smartcar.ino

```diff
@@ -430,6 +431,16 @@ void incomingAvoidanceThreshold(long frontUltDis, long backUltDis)
     }
 }
 
+
+//allows vehicle to avoid objects on its side
+void directionalAvoidance(long leftUltDis,long rightUltDis){
```

Comment:
This logic will make the car drive indefinitely if it's standing next to a wall for example.